safe/report/extractors/general_report.py

```diff
@@ -108,11 +109,25 @@ def general_report_extractor(impact_report, component_metadata):
             hazard_stats.append(stats)
 
         # find total field
-        try:
-            field_name = analysis_inasafe_fields[total_field['key']]
+        fields = [total_not_exposed_field, total_field]
+
+        # calculate the total exposed number by subtract total with
+        # total not exposed number
+        value_to_count = {}
+        for field in fields:
+            field_name = analysis_inasafe_fields[field['key']]
             total = value_from_field_name(field_name, analysis_layer)
+            value_to_count[field['key']] = total
+
+        total_exposed = abs(
```

Comment:
Why do you need to use `abs` here? Should it be always positive if total > total not exposed?. I suggest to remove the `abs`, so that if we got minus number, we can see directly if there is an error.

Reply:
thanks, but we remove the calculation process in the reporting side now, we are using new `total_exposed` table from the analysis layer